package.json

```diff
@@ -191,8 +191,8 @@
         "rimraf": "^3.0.2",
         "rollup": "^3.19.1",
         "sinon": "^14.0.0",
-        "storybook": "^7.0.18",
-        "storybook-builder-wds": "^0.0.1",
+        "storybook": "^7.0.25",
+        "storybook-builder-wds": "https://github.com/bashmish/storybook-builder-wds.git#temp/build",
```

Comment:
I made this temp branch and not gonna touch it unless you need an update or until I release it on NPM

Reply:
release 0.0.2, so switched to that one